zl41arm/board/include/isix_config.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * =====================================================================================
+ *
+ *       Filename:  config.h
```

Comment:
Doesn't match the filename. Maybe remove these headers altogether? They don't bring any useful info anyway.

Reply:
fixed